### ISSUE TITLE
get rid of remaining `#imageLiteral`

### ISSUE THREAD

**Issue by r10s** [deltachat/deltachat-ios#2248]
`#imageLiteral(resourceName:)` has 16 occurrences in our source while the equivalent `UIImage(named:)` has 54.

it does not make much sense to mix these two approaches, also there are some disadvantages in `#imageLiteral(resourceName:)`:

-  `#imageLiteral` make it hard in xcode to see what is going on: they are replaced by some icon, so you cannot easily see the name of the image
- searching a bit around, there seem to be more xcode issues with `#imageLiteral` we did not encounter yet :)

so, to sum up, i do not see a single reason for `#imageLiteral`, let's get rid of that - also to not manifest things by copy+paste as at https://github.com/deltachat/deltachat-ios/pull/2245
